pkg/container-disk/container-disk.go

```diff
@@ -164,6 +172,29 @@ func NewSocketPathGetter(baseDir string) SocketPathGetter {
 	}
 }
 
+func NewBindMountNeededDetector(baseDir string) BindMountNeededDetector {
```

Comment:
I think it would be better to have this function live somewhere in pkg/virt-handler/container-disk
It will be clear that it is being executed in the virt-handler context  - while pkg/container-disk is generic.

Reply:
also there we could reuse some of the function from the pkg/virt-handler/cmd-client/client.go

Reply:
Done